examples/c/CMakeLists.txt

```diff
@@ -46,6 +59,9 @@ find_package(BpfObject REQUIRED)
 
 # Create an executable for each application
 file(GLOB apps *.bpf.c)
+if(NOT CARGO_EXISTS)
+  list(REMOVE_ITEM apps ${CMAKE_CURRENT_SOURCE_DIR}/profile.bpf.c)
+endif()
```

Comment:
do you think it's possible to have a list of apps that require blazesym and do all these modification for them generically? It's fine for now, but thinking ahead, if we add another blazesym example, Cmake will break again. Just adding an app name to some list seems like the best way to keep Cmake working properly